app/models/variant_override.rb

```diff
@@ -6,7 +6,8 @@ class VariantOverride < ActiveRecord::Base
   belongs_to :hub, class_name: 'Enterprise'
   belongs_to :variant, class_name: 'Spree::Variant'
 
-  validates_presence_of :hub_id, :variant_id
+  validates :hub_id, presence: true
+  validates :variant_id, presence: true
```

Comment:
you can have them in a single line as shown in https://guides.rubyonrails.org/v3.2/active_record_validations_callbacks.html#presence